content/capabilities/single-use.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Single Use Object Capabilities"
+section: "Capabilities"
+menu:
+  toc:
+    parent: "capabilities"
+    weight: 10
+---
+## Problem
+
+As shown in the [tutorial page about object capabilities](https://tutorial.ponylang.io/object-capabilities/object-capabilities.html), we can limit the actions of other objects or actors with object capabilities. This is used in the standard library of Pony for network connections and file access, for example. But it can also be used for other systems created in Pony.
+
+For example, let's say we want to implement a capability-restricted service, that returns one unique number every time it is called. We will use a `CustomAuth` as a token to restrict its access, created when provided `AmbientAuth`:
+
+```pony
+use "promises"
+
+primitive CustomAuth
+  new create(auth: AmbientAuth) => None
+
+actor RestrictedService
+  var current_count: USize = 1
+
+  be apply(auth: CustomAuth, promise: Promise[USize]) =>
+    promise(current_count = current_count + 1)
+```
+
+Our `Main` actor receives the `AmbientAuth` token on creation from `env.root`, which means only it or something it provided with that capability can create a new `CustomAuth`. We can then hand out these `CustomAuth` tokens to other actors or objects that need to call our `RestrictedService`.
```

Comment:
> that capability can create a new `CustomAuth`. We can then hand out these `CustomAuth` tokens

I'm a bit uncomfortable with the plural language here, both with the "create a new" and "hand out these tokens" phrasing. I understand the former is heavily tied to the fact that the function you call to gain access to the `CustomAuth` is called `new create`, but I already believe this to be a bit of a misnomer - there is only ever one instance of a primitive, and the fact that we call `new create` to gain access to it is... kind of a weird artifact of the language, IMO.

I bring this up because I think talking about plural `CustomAuth` tokens is likely to confuse the reader as we try to direct their attention to the differences between the `primitive` pattern of object capabilities and the `class` pattern discussed as the main topic of this document.